lib/util/DrilldownUtil.js

```diff
@@ -26,4 +26,25 @@ export function planeId(element) {
  */
 export function asPlaneId(string) {
   return string + planePostfix;
+}
+
+/**
+ * Returns wether the given element is a plane.
+ *
+ * @param  {djs.model.Base|ModdleElement} element
+ * @returns {Boolean}
+ */
+export function isPlane(element) {
+  var di = getDi(element);
+  return is(di, 'bpmndi:BPMNPlane');
+}
+
+/**
+ * Returns the ID of the primary Shape for a plane.
+ *
+ * @param  {djs.model.Base|ModdleElement} element
+ * @returns {String}
+ */
+export function primaryShape(element) {
+  return element.id.replace(planePostfix, '');
```

Comment:
If my element's ID is `_planeCrash_plane` I end up with `Crash_plane` instead of `_planeCrash`. Let's rename `postfix` to `suffix` (much more common) and make sure we always replace the suffix.

Reply:
> Let's rename `postfix` to `suffix`

Yes! Suffix was the word I was looking for! Sometimes I just can't English :sweat_smile:

Reply:
Fixed in 4bb69aaa7d4d5a74b947ebd7225de13109bff581